projects/1.18-fabric/assets/croptosis/croptosis/lang/zh_cn.json

```
@@ -1 +1,40 @@
-{}
\ No newline at end of file
+{
+    "block.croptosis.fertilized_sand": "肥沃沙子",
+    "block.croptosis.fertilized_dirt": "肥沃泥土",
+    "block.croptosis.fertilized_farmland": "肥沃耕地",
+    "block.croptosis.potash_ore": "钾矿石",
```

```suggestion
    "block.croptosis.potash_ore": "钾盐矿石",
```
钾是potassium，建议换成`钾盐`

---

已改